# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2025-07-23 01:41+0000\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

#: src/processes/messages/template.py:5
msgid "You can't pass \"snooze\" for the first task."
msgstr "Нельзя передать «snooze» для первой задачи."

#: src/processes/messages/template.py:7
msgid "You should set task performers before setting template to active."
msgstr "Нужно указать исполнителей задач перед там как активировать шаблон."

#: src/processes/messages/template.py:9
msgid "One or more performers are repeated."
msgstr "Один или несколько исполнителей повторяются."

#: src/processes/messages/template.py:11
msgid "Some fields referenced in the conditions of task \"{name}\" do not exist."
msgstr ""

#: src/processes/messages/template.py:14
msgid "Selections can't be empty on radio or checkbox fields."
msgstr "Значения в полях радио или чекбоксов не могут быть пустыми."

#: src/processes/messages/template.py:15
msgid "A field with a type 'User' should be required."
msgstr "Поле с типом \"Пользователь\" должно быть обязательным."

#: src/processes/messages/template.py:16
msgid "Workflow template name is empty."
msgstr "Не задано имя шаблона."

#: src/processes/messages/template.py:17
msgid "Some of fields in \"workflow template name\" don't exist."
msgstr "Некоторые поля в \"workflow template name\" не существуют."

#: src/processes/messages/template.py:19
msgid "If a \"workflow template name\" contains only an output fields, at least one of the fields must be required."
msgstr ""

#: src/processes/messages/template.py:22
msgid "Kickoff form data not provided."
msgstr "Не предоставлены входные данные процесса."

#: src/processes/messages/template.py:23
msgid "Kickoff form 'id' has incorrect value."
msgstr "Переданно неправильное значение поля 'id' для формы входных данных процесса."

#: src/processes/messages/template.py:24
msgid "Kickoff 'id' not provided."
msgstr "Не предоставлено поле 'id' для формы входных данных процесса."

#: src/processes/messages/template.py:25
msgid "Tasks data not provided, one task required."
msgstr "Данные о задачах не предоставлены, нужна, как минимум, одна задача."

#: src/processes/messages/template.py:26
msgid "Incorrect tasks order."
msgstr "Неправильный порядок задач."

#: src/processes/messages/template.py:27
msgid "You should set 'template owners' value."
msgstr "Необходимо указать 'владельцев шаблона'."

#: src/processes/messages/template.py:28
msgid "You cannot remove yourself from template owners."
msgstr "Вы не можете удалить себя из владельцев шаблона."

#: src/processes/messages/template.py:29
msgid "One or more template owners are incorrect."
msgstr "Один или несколько владельцев шаблона неверны."

#: src/processes/messages/template.py:30
msgid "\"Public success url\" is an invalid format."
msgstr "Поле \"Публичный URL успешного запуска процесса\" имеет недопустимый формат."

#: src/processes/messages/template.py:32
msgid "Filters 'with_tasks_in_progress' and 'workflows_status' cannot be used at the same time."
msgstr "Фильтры 'with_tasks_in_progress' и 'workflows_status' не могут быть использованы одновременно"

#: src/processes/messages/template.py:35
msgid "Permission denied. You are not a template owner."
msgstr "Доступ запрещен. Вы не являетесь владельцем шаблона."

#: src/processes/messages/template.py:37
msgid "Inconsistent permission! Should use only for \"workflow detail\" actions."
msgstr "Противоречие в правах доступа! Следует использовать только для запросов к конкретному процессу."

#: src/processes/messages/template.py:39
msgid "The limit for creating a public id has been exceeded."
msgstr "Превышен лимит на создание \"публичного идентификатора\" шаблона"

#: src/processes/messages/template.py:40
msgid "The limit for creating a embed id has been exceeded."
msgstr "Превышен лимит на создание \"встраиваемого идентификатора\" шаблона"

#. Translators: Template due date editor
#: src/processes/messages/template.py:42
msgid "Expected date field api_name."
msgstr "Ожидается api_name поля типа дата."

#. Translators: Template due date editor
#: src/processes/messages/template.py:44
msgid "Only existing date fields can be used as a due date."
msgstr "В качестве срока выполнения можно использовать только существующие поля даты."

#. Translators: Template due date editor
#: src/processes/messages/template.py:46
msgid "Expected task api_name."
msgstr "Ожидается api_name задачи."

#: src/processes/messages/template.py:47
msgid "Only previous tasks are allowed in a rule."
msgstr "В правиле допускаются только предыдущие задачи."

#: src/processes/messages/template.py:48
msgid "Only previous and current tasks are allowed in a rule."
msgstr "В правиле допускаются только предыдущие и текущие задачи."

#: src/processes/messages/template.py:51
msgid "You should set the user id for performer with the type \"user\"."
msgstr "Вы должны установить идентификатор пользователя для исполнителя с типом \"user\""

#: src/processes/messages/template.py:53
msgid "Performer \"id\" should be a number."
msgstr "Идентификатор исполнителя \"id\" должен быть числом."

#: src/processes/messages/template.py:54
msgid "One or more performers are incorrect."
msgstr "Один или несколько исполнителей неверны."

#: src/processes/messages/template.py:56
msgid "A template with \"Workflow starter\" in the list of performers can not be shared. Please replace the \"workflow starter\" with another team member."
msgstr "Шаблон с «Инициатором процесса» в списке исполнителей не может быть общедоступным. Пожалуйста, замените «инициатора процесса» другим членом команды."

#: src/processes/messages/template.py:61
msgid "You should set the field api_name for performer with the type \"field\"."
msgstr "Вы должны установить имя поля api для исполнителя с типом \"field\""

#: src/processes/messages/template.py:65
msgid "Some fields referenced in the description of task \"{name}\" do not exist."
msgstr "Некоторые поля, указанные в описании задачи \"{name}\", не существуют."

#: src/processes/messages/template.py:71
msgid "Some fields in task \"{name}\" do not exist."
msgstr "Некоторые поля в задаче \"{name}\" не существуют."

#: src/processes/messages/template.py:76
msgid "Task \"{name}\": If the task name contains only output fields, at least one field must be required."
msgstr "Задача \"{name}\": если имя задачи содержит только выходные поля, по крайней мере одно из них должно быть обязательным."

#: src/processes/messages/template.py:81
msgid "Checklist items not exists or invalid."
msgstr "Пункты чек-листа не существуют или указанны некорректно."

#: src/processes/messages/template.py:84
msgid "You can't change api_name of an existing object. Old value: \"{old}\", new value: \"{new}\"."
msgstr "Вы не можете изменить api_name существующего объекта. Старое значеие: \"{old}\", новое значение: \"{new}\""

#: src/processes/messages/template.py:93
msgid "Task \"{task}\": user id \"{user_id}\" doesn't exist or can't be used in this condition."
msgstr "Задача \"{task}\": id пользователя \"{user_id}\" не существует или не может быть использовано в этом правиле."

#: src/processes/messages/template.py:102
msgid "Task \"{task}\": operator \"{operator}\" can't be use with type of field \"{field_type}\"."
msgstr "Задача \"{task}\": оператор \"{operator}\" не может быть использован с типом поля \"{field_type}\"."

#: src/processes/messages/template.py:111
msgid "Task \"{task}\": selection \"{selection_api_name}\" doesn't exist or can't be used in this condition."
msgstr "Задача \"{task}\": значение поля \"{selection_api_name}\" не существует или не может быть использовано в этом правиле."

#: src/processes/messages/template.py:119
msgid "Task \"{task}\": operator \"{operator}\" should have some value."
msgstr "Задача \"{task}\": Требуется передать значение для выполнения действия с оператором \"{operator}\"."

#: src/processes/messages/template.py:125
msgid "Task \"{name}\": The checklist contains a duplicate api_name \"{api_name}\". Recreate the checklist or change its api_name."
msgstr "Задача \"{name}\": контрольный список содержит дублирующийся api_name \"{api_name}\". Создайте список заново или измените его api_name."

#: src/processes/messages/template.py:133
msgid "Task \"{name}\": The checklist item contains a duplicate api_name \"{api_name}\". Recreate the item or change its api_name."
msgstr "Задача \"{name}\": элемент контрольного списка содержит дублирующийся api_name \"{api_name}\". Создайте элемент заново или измените его api_name."

#: src/processes/messages/template.py:141
msgid "Task \"{name}\": The condition contains a duplicate api_name \"{api_name}\". Recreate the condition or change its api_name."
msgstr "Задача \"{name}\": условие содержит дублирующийся api_name \"{api_name}\". Создайте условие заново или измените его api_name."

#: src/processes/messages/template.py:149
msgid "Task \"{name}\": Output field \"{field_name}\" contains a duplicate api_name \"{api_name}\". Change the api_name or recreate the output field."
msgstr "Задача \"{name}\": выходное поле \"{field_name}\" содержит дублирующийся api_name \"{api_name}\". Измените api_name или пересоздайте поле."

#: src/processes/messages/template.py:159
msgid "Task \"{name}\": The condition predicate contains a duplicate api_name \"{api_name}\". Recreate the predicate or change its api_name."
msgstr "Задача \"{name}\": предикат условия содержит дублирующийся api_name \"{api_name}\". Создайте предикат заново или измените его api_name."

#: src/processes/messages/template.py:167
msgid "Task \"{name}\": The due date contains a duplicate api_name \"{api_name}\". Recreate the due date or change its api_name."
msgstr "Задача \"{name}\": срок выполнения содержит дублирующийся api_name \"{api_name}\". Создайте срок заново или измените его api_name."

#: src/processes/messages/template.py:175
msgid "Task \"{name}\": The condition rule contains a duplicate api_name \"{api_name}\". Recreate the rule or change its api_name."
msgstr "Задача \"{name}\": правило условия содержит дублирующийся api_name \"{api_name}\". Создайте правило заново или измените его api_name."

#: src/processes/messages/template.py:183
msgid "Task \"{name}\": Option \"{value}\" of output field \"{field_name}\" contains a duplicate api_name \"{api_name}\". Change the api_name or recreate the option."
msgstr "Задача \"{name}\": вариант \"{value}\" выходного поля \"{field_name}\" содержит дублирующийся api_name \"{api_name}\". Измените api_name или пересоздайте вариант."

#: src/processes/messages/template.py:194
msgid "Task \"{name}\": Contains a duplicate api_name \"{api_name}\". Recreate the task or change its api_name."
msgstr "Задача \"{name}\": содержит дублирующийся api_name \"{api_name}\". Создайте задачу заново или измените её api_name."

#: src/processes/messages/template.py:202
msgid "Task \"{name}\": The performer contains a duplicate api_name \"{api_name}\". Recreate the performer or change its api_name."
msgstr "Задача \"{name}\": исполнитель содержит дублирующийся api_name \"{api_name}\". Создайте исполнителя заново или измените его api_name."

#: src/processes/messages/template.py:208
msgid "Duplicate template owners in the add owners request."
msgstr "Дублирующиеся владельцы шаблона в запросе на добавление владельцев."

#: src/processes/messages/template.py:211
msgid "Template \"{name}\": The owners contain a duplicate api_name \"{api_name}\". Change the duplicate api_name or specify different owners."
msgstr "Шаблон \"{name}\": владельцы содержат дублирующийся api_name \"{api_name}\". Измените дублирующийся api_name или укажите других владельцев."

#: src/processes/messages/template.py:220
msgid "Task \"{name}\": The \"Return to\" setting refers to a non-existent task (API name: \"{api_name}\")."
msgstr "Задача \"{name}\": параметр \"Возврат к\" ссылается на несуществующую задачу (API name: \"{api_name}\")."

#: src/processes/messages/template.py:227
msgid "Task \"{name}\": Cannot be set to return to itself."
msgstr "Задача \"{name}\": не может быть настроена на возврат к самой себе."

#: src/processes/messages/template.py:231
msgid "Task \"{name}\": Can be returned only to one of its ancestors."
msgstr "Задача \"{name}\": может быть возвращена только к одному из предшественников."

#: src/processes/messages/template.py:236
msgid "Task \"{task}\": Group ID \"{group_id}\" does not exist or cannot be used in this condition."
msgstr "Задача \"{task}\": идентификатор группы \"{group_id}\" не существует или не может быть использован в этом условии."

#: src/processes/messages/template.py:244
msgid "Task condition \"{name}\": Only the \"completed\" operator is allowed for the \"start_task\" action"
msgstr "Условие задачи \"{name}\": Для действия \"start_task\" разрешен только оператор \"completed\""

#: src/processes/messages/template.py:251
msgid "Task \"{name}\" has a circular dependency. Please change the \"начать после\" condition."
msgstr "Задача \"{name}\" попала в циклическую зависимость, измените условие \"начать после\""

#: src/processes/messages/template.py:258
msgid "Task \"{name}\": the \"start after\" condition points to a non-existent task. Please change the \"start after\" condition."
msgstr ""

#. Translators: Number field validation
#: src/processes/messages/template.py:243
#: src/processes/messages/workflow.py:171
msgid "The value must be a number."
msgstr "Значение должно быть числом."

#: src/processes/messages/workflow.py:5
msgid "Saving files is not available, file storage does't configured. Contact support."
msgstr "Невозможно сохранить файл, Файловое хранилище не настроенно. Обратитесь в поддержку."

#: src/processes/messages/workflow.py:8
msgid "Only \"snoozed\" workflows can be resumed."
msgstr "Только «отложенные» процессы могут быть возобновлены"

#: src/processes/messages/workflow.py:9
msgid "You need to resume the workflow before completing the task."
msgstr "Вам нужно возобновить процесс перед выполнением задачи."

#: src/processes/messages/workflow.py:10
msgid "Task is skipped and cannot be completed."
msgstr "Пропущенная задача не может быть завершена."

#: src/processes/messages/workflow.py:12
msgid "Check off all the items in the checklist before completing the task."
msgstr "Отметьте все элементы в чек-листе, прежде чем завершить задачу."

#: src/processes/messages/workflow.py:14
msgid "You have already completed this task."
msgstr "Вы уже завершили эту задачу."

#: src/processes/messages/workflow.py:15
msgid "You have already completed this workflow."
msgstr "Вы уже завершили этот процесс."

#: src/processes/messages/workflow.py:16
msgid "You are not allowed to finish this workflow."
msgstr "У вас нет прав для завершения данного процесса."

#: src/processes/messages/workflow.py:17
msgid "Task already completed."
msgstr "Задача уже завершена."

#: src/processes/messages/workflow.py:18
msgid "You don't have permission for complete the task."
msgstr "У вас нет прав для завершения задачи."

#. Translators: Task return api
#: src/processes/messages/workflow.py:20
msgid "The supplied task number is incorrect."
msgstr "Предоставлен неверный номер задачи."

#. Translators: Create / delete task performer action
#: src/processes/messages/workflow.py:22
msgid "The user with the specified credentials not found."
msgstr "Пользователь с указанными учетными данными не найден."

#: src/processes/messages/workflow.py:25
msgid "You have reached the limit of the number of guest performers."
msgstr "Вы достигли лимита числа гостей."

#. Translators: Create / delete task performer action
#: src/processes/messages/workflow.py:28
msgid "Deleting the last performer is not allowed."
msgstr "Последний исполнитель не может быть удален."

#: src/processes/messages/workflow.py:29
msgid "Completed workflow cannot be changed."
msgstr "Завершенный процесс не может быть изменен."

#. Translators: Create / delete task performer or fill checklist actions
#: src/processes/messages/workflow.py:31
msgid "Available only for the active task."
msgstr "Доступно только для активной задачи."

#: src/processes/messages/workflow.py:33
msgid "You need to be added to the task as a performer to check off this checklist item."
msgstr "Вы должны быть добавлены к задаче как исполнитель, чтобы отметить этот элемент чек-листа."

#: src/processes/messages/workflow.py:37
msgid "You need to resume the workflow before check off this checklist item."
msgstr "Вам нужно возобновить процесс, прежде чем отмечать этот элемент чек-листа."

#: src/processes/messages/workflow.py:39
msgid "Permission denied."
msgstr "Доступ запрещен."

#. Translators:
#: src/processes/messages/workflow.py:42
msgid "Please fill in the required fields."
msgstr "Пожалуйста, заполните обязательные поля."

#: src/processes/messages/workflow.py:44
msgid "Wrong argument format: \"{arg}\"."
msgstr "Неверный формат аргумента: \"{arg}\"."

#. Translators: String field validation
#: src/processes/messages/workflow.py:48
msgid "Value should be a string."
msgstr "Значение должно быть строкой."

#: src/processes/messages/workflow.py:51
msgid "The value of the field exceeds the limit of {limit} characters."
msgstr "Значение поля превысило лимит в {limit} символов."

#. Translators: Dropdown / radio / checkbox selections validation
#: src/processes/messages/workflow.py:55
msgid "Selection with the given id does not exist."
msgstr "Значение поля с указнным id не существует."

#. Translators: Checkbox selections validation
#: src/processes/messages/workflow.py:57
msgid "Checkbox value should be a list."
msgstr "Значение поля \"чекбокс\" должно быть списоком."

#. Translators: Checkbox selections validation
#: src/processes/messages/workflow.py:59
msgid "Checkbox value contains invalid selections."
msgstr "В поле \"чекбокс\" указазны некорректные значения"

#. Translators: Checkbox selections validation
#: src/processes/messages/workflow.py:61
msgid "Checkbox value contains non existent selections."
msgstr "В поле \"чекбокс\" выбраны несуществующие значения."

#: src/processes/messages/workflow.py:64
msgid "The date value must be an integer or a floating-point number."
msgstr "Значение даты должно быть целым числом или с плавающей запятой."

#. Translators: URL field validation
#: src/processes/messages/workflow.py:67
msgid "URL field value should be a string."
msgstr "Значение в поле \"URL\" должно быть строкой."

#. Translators: URL field validation
#: src/processes/messages/workflow.py:69
msgid "The URL is invalid."
msgstr "Указан некорректный URL."

#. Translators: File field validation
#: src/processes/messages/workflow.py:71
msgid "File value should be a list of integers."
msgstr "Значение поля \"Файл\" должно быть массивом целых чисел."

#. Translators: File field validation
#: src/processes/messages/workflow.py:73
msgid "Attachment not found."
msgstr "Вложение не найдено."

#. Translators: User field validation
#: src/processes/messages/workflow.py:75
msgid "The value must be the ID of an existing account user."
msgstr "Значение должно быть идентификатором существующего пользователя."

#: src/processes/messages/workflow.py:77
msgid "Save attachments failed. Service unavailable."
msgstr "Сохранение вложения невозможно. Сервис недоступен."

#: src/processes/messages/workflow.py:78
msgid "Blob object doesn't exist."
msgstr "Объект \"Blob\" не существует."

#: src/processes/messages/workflow.py:79
msgid "AI service is temporarily unavailable."
msgstr "ИИ сервис временно недоступен."

#. Translators: AI template generation
#: src/processes/messages/workflow.py:81
msgid "Could not create template from given description."
msgstr "Не удалось создать шаблон из данного описания."

#: src/processes/messages/workflow.py:83
msgid "You've reached the limit for free AI powered template generation requests, sign up for a premium account to get unlimited access."
msgstr "Вы достигли лимита бесплатных запросов на генерацию шаблонов с помощью ИИ, подпишитесь на премиум-аккаунт для получения неограниченного доступа."

#: src/processes/messages/workflow.py:89
msgid "Something seems to be wrong with your input, please try re phrasing your process name and description."
msgstr "Что-то не так с вашим промытом, пожалуйста, попробуйте переформулировать название и описание вашего процесса"

#: src/processes/messages/workflow.py:94
msgid "You need to set up a system prompt for template generation. Contact administrator."
msgstr "Необходимо настроить системный запрос для генерации шаблона. Свяжитесь с администратором."

#. Translators: Comment create validation
#: src/processes/messages/workflow.py:98
msgid "You need to specify the comment text or attachments."
msgstr "Необходимо указать текст или вложение комментария."

#: src/processes/messages/workflow.py:99
msgid "You cannot write a comment on a completed workflow."
msgstr "Вы не можете написать комментарий к завершенному процессу."

#: src/processes/messages/workflow.py:100
msgid "You cannot change a deleted comment."
msgstr "Вы не можете изменить удаленный комментарий."

#: src/processes/messages/workflow.py:102
msgid "The size limit for attachments is {mb} Mb."
msgstr "Лимит размера вложений составляет {mb} Мб."

#. Translators: Set task / workflow due date
#: src/processes/messages/workflow.py:106
msgid "Due date should be greater than current."
msgstr "Дата выполнения должна быть больше текущей."

#. Translators: Starting workflow. Validate user field
#: src/processes/messages/workflow.py:108
msgid "Field with type \"User\" should be required."
msgstr "Поле с типом \"User\" должно быть обязательным."

#: src/processes/messages/workflow.py:111
msgid "The default value for a field with type \"User\" should be an integer."
msgstr "Значение по умолчанию для поля с типом \"User\" должно быть целым числом."

#: src/processes/messages/workflow.py:113
msgid "The user for a field with type \"User\" does not exist."
msgstr "Не найден пользователь указанный в поле с типом \"User\"."

#. Translators: Starting workflow. Validate dropdown / radio / checkbox
#: src/processes/messages/workflow.py:115
msgid "Field selections not provided."
msgstr "Элементы поля не указаны."

#: src/processes/messages/workflow.py:116
msgid "Checklist element length limit exceeded. Limit 2000 chars."
msgstr "Превышен лимит длины элемента чек-листа. Лимит 2000 символов."

#. Translators: Task list filter by performer
#: src/processes/messages/workflow.py:118
msgid "Filter by \"assigned_to\" is only allowed for the admin."
msgstr "Фильтр по \"assigned_to\" разрешен только для роли администратора."

#: src/processes/messages/workflow.py:119
msgid "Checklist item with given id not found."
msgstr "Элемент чек-листа с данным идентификатором не найден."

#. Translators: System template creation validation
#: src/processes/messages/workflow.py:121
msgid "You can't pass the `id` field."
msgstr "Нельзя применять поле \"id\"."

#. Translators: System template creation validation
#: src/processes/messages/workflow.py:123
msgid "Radio and checkbox fields must have non-empty selections."
msgstr "Поля с радиокнопками и флажками должны иметь непустые значения."

#. Translators: System template kickoff creation validation
#: src/processes/messages/workflow.py:125
msgid "Value should be a object."
msgstr "Значение должно быть объектом."

#. Translators: System template fields creation validation
#: src/processes/messages/workflow.py:127
msgid "Value should be a list."
msgstr "Значение должно быть списком."

#. Translators: System template tasks creation validation
#: src/processes/messages/workflow.py:129
msgid "Incorrect order of tasks."
msgstr "Неверный порядок задач."

#. Translators: System template creation validation
#: src/processes/messages/workflow.py:131
msgid "You must pass an \"api_name\" into every field you declare."
msgstr "Вы должны передать 'api_name' в каждое поле, которое вы объявляете."

#: src/processes/messages/workflow.py:133
msgid "Your template is disabled. Please, enable this template before running."
msgstr "Ваш шаблон отключен. Пожалуйста, включите этот шаблон перед запуском."

#: src/processes/messages/workflow.py:136
msgid "Only \"in-progress\" workflows can be filtered by current performer."
msgstr "Только процессы в статусе \"в процессе\" могут быть отфильтрованы по текущему исполнителю."

#: src/processes/messages/workflow.py:139
msgid "The task can't be completed until all the embedded workflows are done."
msgstr "Задача не может быть завершена до завершения всех встроенных процессов."

#: src/processes/messages/workflow.py:142
msgid "The task can't be returned until all the embedded workflows are done."
msgstr "Задача не может быть возвращена, пока все встроенные рабочие процессы не завершены."

#: src/processes/messages/workflow.py:144
msgid "Snoozed workflow cannot be changed."
msgstr "Отложенный рабочий процесс не может быть изменен."

#: src/processes/messages/workflow.py:145
msgid "Completed task cannot be changed."
msgstr "Выполненная задача не может быть изменена."

#: src/processes/messages/workflow.py:147
msgid "Embedded workflows can be created only for the workflow's active task."
msgstr "Встроенные рабочие процессы можно создавать только для активной задачи рабочего процесса."

#: src/processes/messages/workflow.py:150
msgid "You need to be added to the task as a performer in order to create a embedded workflows."
msgstr "Вам необходимо быть исполнителем в задаче, чтобы создать встроенные процессы."

#: src/processes/messages/workflow.py:153
msgid "You need to pass in a \"task_id\", or a \"task api_name\"."
msgstr "Необходимо передать task \"id\" или \"api_name\" задачи"

#: src/processes/messages/workflow.py:154
msgid "The passed in task was not found in the workflow."
msgstr "Указанная задача не найдена среди задач процесса."

#: src/processes/messages/workflow.py:155
msgid "You can't revert from the first task."
msgstr "Нельзя вернуться с первой задачи."

#: src/processes/messages/workflow.py:157
msgid "Unable return to \"{task_name}\" because it was skipped by a condition."
msgstr "Нелья вернутья к задаче \"{task_name}\", т.к задача пропускается по условию."

#: src/processes/messages/workflow.py:162
msgid "Unable return to \"{task_name}\", because it was skipped by a condition (as were all the tasks before it)."
msgstr "Нельзя вернутья к задаче \"{task_name}\", т.к задача пропускается по условию (и все предыдущие задачи тоже)."

#: src/processes/messages/workflow.py:167
msgid "A workflow can't return to a later task."
msgstr "Нельзя вернуться к одной из будущих задач."

#: src/processes/messages/workflow.py:168
msgid "There is no group with the specified ID."
msgstr "Группа с указанным ID не найдена."

#: src/processes/messages/workflow.py:169
msgid "You must supply a comment."
msgstr "Необходимо оставить комментарий."

#. Translators: task actions validation
#: src/processes/messages/workflow.py:173
msgid "Task not found."
msgstr "Задача не найдена."

#: src/processes/messages/workflow.py:174
#, fuzzy
#| msgid "Available only for the active task."
msgid "Available only for active tasks."
msgstr "Доступно только для активной задачи."

#: src/processes/messages/workflow.py:176
msgid "You must be added to the task as a performer to complete this action."
msgstr "Чтобы выполнить это действие, вас нужно добавить к задаче как исполнителя."

#: src/processes/messages/workflow.py:178
msgid "Completed task can't be returned."
msgstr "Завершённую задачу нельзя вернуть."

#: src/processes/messages/workflow.py:179
msgid "You cannot write a comment on a inactive task."
msgstr "Нельзя написать комментарий к неактивной задаче."

#: src/processes/messages/workflow.py:181
msgid "The user or group specified for a \"User\" field does not exist."
msgstr "Указанный пользователь или группа для поля типа \"Пользователь\" не существует."

#: src/processes/messages/template.py:247
msgid "Task \"{task}\": Group ID \"{group_id}\" does not exist or cannot be used in this condition."
msgstr "Задача «{task}»: идентификатор группы «{group_id}» не существует или не может быть использован в этом условии."

#: src/processes/messages/workflow.py:197
msgid "The user or group for a field with type \"User\" does not exist."
msgstr "Указанный пользователь или группа для поля типа «Пользователь» не существует."
